Lookup a container's IP address individually during healthcheck URI

scooterGrun opened the pull request:
Hi Alex,

We ran into a timing issue whereby if there were two containers that contained URI healthchecks, the plugin would fail due to the IPAddresses() call during the first healthcheck. This would attempt to find ALL the container IPs, however at that point the second container hasnt yet started and therefore doesnt have a valid IP (yet).

I've submitted a very small fix for it, whereby the URI healthcheck looks up only the IP of the container its starting.

alexec commented:
Great stuff. Do you think a unit test would be a good idea? I like to have them as it makes sure that future changes don't break fixes. For example, it might be good to have one that checks an exception is throw when ports are not exposed.

scooterGrun commented:
Hi Alexe,

I've added some unit tests around the new method, and switched the docker exec to use the containerId rather than containerName.

Thanks
